dbt/models/default/schema/default.vw_pin_appeal.yml

```diff
@@ -59,7 +65,7 @@ unit_tests:
           - {parid: "123", taxyr: "2024", cur: "Y", deactivat: null, class: "2.1-1)A"}
       - input: source("iasworld", "htpar")
         rows:
-          - {parid: "123", taxyr: "2024", cur: "Y", deactivat: null, cpatty: "1", caseno: "1"}
+          - {parid: "123", taxyr: "2024", cur: "Y", deactivat: null, cpatty: "1", caseno: "1", heartyp: "A", user125: "A", user38: "CC", user104: "A", resact: "A", hrstatus: "A", user100: "A", user101: "A", user89: "A"}
```

Comment:
@jeancochrane this works, but I'm not sure if I've included extraneous columns.

Reply:
It should work with only `heartyp`, since that's the only new column that's being used in a table filter or join. The rest of the new columns here after `heartyp` should be unnecessary. Does that work as expected?